stages/org.osbuild.pacman.conf

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+"""
+configure pacman
+"""
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": true
+"""
+
+
+def main(tree, options):
+    cfg = """
+[options]
+Architecture = x86_64
```

Comment:
Would probably be better if we are arch independent?

Reply:
Make it a config options for the stage?

Reply:
From the man page:

```
       Architecture = auto &| i686 &| x86_64 | ...
           If set, pacman will only allow installation of packages with the given architectures (e.g.  i686, x86_64, etc). The special value auto
           will use the system architecture, provided via “uname -m”. If unset, no architecture checks are made.  NOTE: Packages with the special
           architecture any can always be installed, as they are meant to be architecture independent.
```

So I am not 100% sure if we can use auto, in theory Arch does not support anything else then x86_64 but some derivates do.

Reply:
Could make it an option with a default value? like `options.get("architecture", "x86_64")`?

Reply:
For sure!